docs/maven-bnd.md

```diff
@@ -0,0 +1,109 @@
+# Migration to a Maven + Bnd based build system
+
+We are in the process of changing the build system.
+We are not accepting new pull requests that are still using the old way from here on.
+
+## Background
+
+openHAB is a java project that is build around the OSGi framework. openHAB addons are OSGi bundles.
+
+Over the years the [bnd software](https://bnd.bndtools.org/) ("bnd is the engine behind a number of popular software development tools that support OSGi") has established itself as the defacto standard for creating OSGi bundles.
+
+The openHAB maintainers have decided to move from the very Eclipse centric buildsystem to maven/bnd. A combination that not only allows to develop bindings seamlessly in other IDEs than Eclipse, but Java beginners and intermediates find a commonly known maven buildsystem.
+
+## Advantages of the new buildsystem 
+
+* Builds faster
+* Offline mode builds
+* Less buildsystem file overhead
+* External dependencies per extension via the standard maven dependency system
+* Unit tests are now part of the same project (Eclipse allows to create a unit test for a class via GUI now)
+
+## How to migrate
+
+1. Move your addon to `/bundles/`. Adapt the `addons/pom.xml` and `/bundles/pom.xml`
+2. Copy over the `.classpath` from an already migrated bundle
+2. Copy over `.project` and adapt the `name`.
```

Comment:
Two times "2."

Reply:
I agree that this should be fixed, but Github does not care about the number at all (IIRC). It uses an auto enumeration.

Regardless of Github's logic, I will change it.

Reply:
done